kornia/enhance/integral.py

```diff
@@ -0,0 +1,74 @@
+from typing import Tuple
+
+from kornia.core import Module, Tensor
+from kornia.testing import KORNIA_CHECK_SHAPE
+
+
+def integral_tensor(input: Tensor, dim: Tuple[int] = None) -> Tensor:
+    r"""Calculates integral of the input tensor.
+
+    Args:
+        input: the input tensor with shape :math:`(B,C,H,W)` with shape :math:`(B,C,H,W)`.
+
+    Returns:
+        Integral tensor for the input tensor
+
+    Examples:
+        >>> input = torch.randn(2,2,5,5)
+        >>> output = integral(input)
+        >>> output.shape
+        torch.Size([2, 2, 5, 5])
+    """
+    if not isinstance(input, Tensor):
+        raise TypeError(f"Input type is not a torch.Tensor. Got {type(input)}")
+    KORNIA_CHECK_SHAPE(input, ["B", "C", "H", "W"])
+    if dim is None:
+        dim = (-1,)
+    S: Tensor = input
+    for i in dim:
+        S = S.cumsum(i)
+    return S
+
+
+def integral_image(image: Tensor) -> Tensor:
+    r"""Calculates integral of the input image tensor.
+
```

Comment:
Can you a one line description here that explains the difference between the othe one ? For example:
"Sum values along the image rows and columns"